docs/products/opensearch/howto/sample-dataset.rst

```diff
@@ -142,15 +139,12 @@ And the mapping with the fields and their respective types.
     'protein': {'type': 'float'},
     'rating': {'type': 'float'},
     'sodium': {'type': 'float'},
-    'title': {'fields': {'keyword': {'ignore_above': 256,
-                                    'type': 'keyword'}},
+    'title': {'fields': {'keyword': {'ignore_above': 256, 'type': 'keyword'}},
             'type': 'text'}}
 
-
-
 .. seealso::
 
-    More about `OpenSearch mapping <https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/create-index/#mappings>`_ 
+    Read more about in the `Official OpenSearch Documentation <https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/create-index/#mappings>`_.
```

Comment:
This reads a bit wrong - did you mean "Read more about it" Also, I don't think we need to use capital case here. so it should be something like this -

Read more about dynamic mapping in the `official OpenSearch documentation <https://opensearch.org/docs/latest/opensearch/rest-api/index-apis/create-index/#mappings>`_.

Reply:
thanks, fixed!